docker/utils/utils.py

```diff
@@ -615,6 +615,16 @@ def create_container_config(
         if volumes_from is not None:
             raise errors.InvalidVersion(message.format('volumes_from'))
 
+    # NetworkMode must be present and valid in host config from 1.20 onwards
```

Comment:
It feels like that snippet of code belongs to `create_host_config` instead. What's the rationale for having it here?